index.html

```diff
@@ -0,0 +1,370 @@
+                fill="#505F79"
+              />
+            </svg>
+          </a>
+        </li>
+      </ul>
+      <button class="aboutbutton">Get my Resume</button>
+        <ul class="skillset">
+          <li>
+            <h2>languages</h2> <img class="arrow" src="./images/down.png" alt="rightarrow" />
+          </li>
+          <li class="lang">
+            <img src="./images/language3.png" alt="js" />
+            <p>Javascript</p>
+          </li>
+          <li class="lang">
+            <img src="./images/language1.png" alt="js" />
+            <p>HTML</p>
+          </li>
+          <li class="lang">
+            <img src="./images/language2.png" alt="js" />
+            <p>CSS</p>
+          </li>
+          <li><h2>Frameworks </h2><img class="arrow" src="./images/right.png" alt="rightarrow" /></li>
+          <li><h2>Skills </h2><img class="arrow" src="./images/right.png" alt="rightarrow" />
```

Comment:
- [ ] There are two levels in the grouping here.  The `Languages, JavaScript, ..., Frameworks, Skills` are not in the same level of grouping. Rather, `Languages` groups `JavaScript, HTML, CSS` within it. You can the following as an example
```
<ul> 
   <li> Languages </>
      <ul>
        <li> JavaScript </li>
        <li> HTML </li>
      </ul>
   </li>
   <li> Frameworks </li>
</ul>
```